src/octokit/get-webhooks.ts

```diff
@@ -1,16 +1,17 @@
-import { Webhooks } from "@octokit/webhooks";
+import { WebhookEvent, Webhooks } from "@octokit/webhooks";
 
 import { State } from "../types";
 import { getErrorHandler } from "../helpers/get-error-handler";
 import { webhookTransform } from "./octokit-webhooks-transform";
 
+import { Context } from "../context";
+
 export function getWebhooks(state: State) {
-  const webhooks = new Webhooks({
+  const webhooks = new Webhooks<WebhookEvent, Pick<Context, "github" | "log">>({
```

Comment:
@ankeetmaini sorry I missed this, but I wonder why picking just `github` and `log` here? I've run into another regression because `context.issue` is not defined:

https://github.com/probot/create-probot-app/runs/1227162158?check_suite_focus=true#step:4:206

Reply:
I am sorry, I saw this comment and didn't see any other usages apart from the above. https://github.com/probot/probot/blob/630d78e34780c8f64702a15b42854c2fa0f9af54/src/context.ts#L53

Reply:
I am sorry, I saw this comment and didn't see any other usages apart from the above. https://github.com/probot/probot/blob/630d78e34780c8f64702a15b42854c2fa0f9af54/src/context.ts#L53

Reply:
I'll raise a PR immediately

Reply:
I just did: https://github.com/probot/probot/pull/1378 sorry I hope you'll see this in time